style.css

```diff
@@ -0,0 +1,268 @@
+.body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+}
+
+.header::before {
+  content: '';
+  position: absolute;
+  width: 461px;
+  height: 1800px;
+  background: #000;
+ }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)